Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -161,6 +162,19 @@ open class NavigationMapView: UIView {
         }
     }
     
+    var simulation: Bool = true
```

Comment:
This property’s name should make clear that it’s a Boolean flag (e.g., “simulatesLocation”). It‘s also only applicable to two of the possible value of `puckType`, so it should be an associated value instead of a parallel property.

Reply:
Thanks for the suggestion, I'll rename it to `simulatesLocation `. This property defaults to `true` to allow the puck receive location update from the routeController. But it doesn't change with the puckType.